src/pybanana/api.py

```
@@ -1,169 +1,184 @@
             return None
 
-    def get_download_url(self, model_name: ModelType, item_id: int) -> Optional[str]:
+    async def get_download_url(
+        self, model_name: ModelType, item_id: int
+    ) -> Optional[str]:
         """Get the download URL for a specific file."""
         try:
-            data = self._get_data(f"/{model_name}/{item_id}/ProfilePage")
+            data = await self._get_data(f"/{model_name}/{item_id}/ProfilePage")
             return data.get("_sDownloadUrl", "")
         except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)